src/IMangrove.sol

```diff
@@ -62,118 +101,334 @@ interface IMangrove is HasMgvEvents {
   function local(OLKey memory olKey) external view returns (Local _local);
 
+  ///@notice Gets the global configuration for Mangrove and local the configuration for a specific offer list.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@return _global The global configuration for Mangrove.
+  ///@return _local The local configuration for the offer list.
   function config(OLKey memory olKey) external view returns (Global _global, Local _local);
 
+  ///@notice Determines whether the reentrancy lock is in effect for the offer list.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@return true if locked; otherwise, false.
+  ///@dev The lock protects modifying or inspecting the offer list while an order is in progress.
   function locked(OLKey memory olKey) external view returns (bool);
 
+  ///@notice The `offerId` of the best offer on the offer list.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@return offerId The `offerId` of the best offer on the offer list.
   function best(OLKey memory olKey) external view returns (uint offerId);
 
+  ///@notice the offer list key given by the hash (if the offer list key has been activated at least once).
+  ///@param olKeyHash the hash of the offer list key.
+  ///@return olKey the olKey.
   function olKeys(bytes32 olKeyHash) external view returns (OLKey memory olKey);
 
   // # Offer view functions
 
+  ///@notice Gets an offer in packed format.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@param offerId The `offerId` of the offer on the offer list.
+  ///@return offer The offer in packed format.
   function offers(OLKey memory olKey, uint offerId) external view returns (Offer offer);
 
+  ///@notice Gets an offer detail in packed format.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@param offerId The `offerId` of the offer on the offer list.
+  ///@return offerDetail The offer detail in packed format.
   function offerDetails(OLKey memory olKey, uint offerId) external view returns (OfferDetail offerDetail);
 
+  ///@notice Gets both an offer and an offer detail in packed format.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@param offerId The `offerId` of the offer on the offer list.
+  ///@return offer The offer in packed format.
+  ///@return offerDetail The offer detail in packed format.
   function offerData(OLKey memory olKey, uint offerId) external view returns (Offer offer, OfferDetail offerDetail);
 
   // # Governance functions
 
+  ///@notice Gets the governance address.
+  ///@return the governance address.
   function governance() external view returns (address);
```

Comment:
Should we annotate the functions that can only be called by governance?

Reply:
The checker enforces all functions to have natspec. So easier to just add "dummy" natspec than improving the tool now :D